File: SofiForce.Models/Models/EntityModels/ApplicationFeaturePermissionModel.cs
using System;

namespace Models
{
    public class ApplicationFeaturePermissionModel
    {
        public int FeaturePermissionId { get; set; }
        public int? FeatueId { get; set; }
        public string FeaturePermissionCode { get; set; }
        public string FeaturePermissionNameEn { get; set; }
        public string FeaturePermissionNameAr { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ApplicationFeaturePermissionModel.ts
import { ApplicationFeatureModel } from "./applicationFeatureModel";
import { AppRoleFeaturePermissionModel } from "./appRoleFeaturePermissionModel";

export interface ApplicationFeaturePermissionModel {
    featurePermissionId: number;
    featueId: number | null;
    featurePermissionCode: string;
    featurePermissionNameEn: string;
    featurePermissionNameAr: string;
    icon: string;
    color: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}